### PR TITLE
Update Action Network

### PR DESCRIPTION
**Pull request by hkamran80**

### CODE REVIEW

File: img/a/actionnetwork.org.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 93.4"><path id="a" d="M20 46 3.3 1.1C2.5-.6 0-.2 0 1.7v90c0 1.8 2.6 2.3 3.2.6l16.7-45a1.7 1.7 0 0 0 0-1z"/><use fill="#fd570d" href="#a"/><use x="20" fill="#CC70EB" href="#a"/><use x="40" fill="#1781FA" href="#a"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg"><path fill="#FD570D" d="M37 85 6 2C5-1 0 0 0 3v166c0 4 5 4 6 1l31-83a3 3 0 0 0 0-2Z" style="--darkreader-inline-fill:#c43d02"/><path fill="#CC70EB" d="M74 85 43 2c-1-3-6-2-6 1v166c0 4 5 4 6 1l31-83v-2Z" style="--darkreader-inline-fill:#661283"/><path fill="#1781FA" d="M111 85 80 2c-1-3-6-2-6 1v166c0 4 5 4 6 1l31-83v-2Z" style="--darkreader-inline-fill:#045abd"/></svg>
```
Comment (Carlgo11) [inline img/a/actionnetwork.org.svg]: Uhm why is this change necessary?
The previous code was both smaller in size and IMHO way cooler :stuck_out_tongue:

Comment (hkamran80) [reply img/a/actionnetwork.org.svg]: Forgot to add the reason to the description! They updated their branding in September. Here's [their blog post](https://actionnetwork.blog/unveiling-action-networks-new-logo-branding/).

Comment (Carlgo11) [reply img/a/actionnetwork.org.svg]: I think #7043 already fixed that :thinking:

Comment (hkamran80) [reply img/a/actionnetwork.org.svg]: Huh, that didn't come up in my PR search. I'll close this and make a new PR with just the 2FA update. Also, the new logo isn't on the site.
